ocaml/sdk-gen/component-test/spec/session.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Better to put this into an example folder. 
(Otherwise, the test IDs, and the `expected_result` have to be more meaningful.)

Reply:
QA has updated this value of expected_result for the actual test scenario in other upcoming PR.

Reply:
Hi @minglumlu , I will update based on your comments in my component test PR. What is more meaningful you mean here? Need some explanation.